### PR TITLE
Feature us9

### PR DESCRIPTION
**Pull request by CananOzyol**

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -44,6 +49,22 @@
...
 			<version>4.6.2</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-webflux</artifactId>
+			<version>6.0.10</version>
+		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-test</artifactId>
```
Comment (ZeshanShahid79) [inline backend/pom.xml:59]: do u need this dependency?

File: backend/src/main/resources/application.properties [hunk 2]
```diff
@@ -1 +1,3 @@
 spring.data.mongodb.uri=${MONGO_DB_URI}
+randomCatImage-api.url=https://api.unsplash.com
+api.unsplash.access.key=${API_UNSPLASH_ACCESS_KEY}
```
Comment (ZeshanShahid79) [inline backend/src/main/resources/application.properties:3]: pls add a new line here

File: backend/pom.xml (hunk 1 above)
Comment (CananOzyol) [reply backend/pom.xml]: apparently I don't :)

File: backend/src/main/resources/application.properties (hunk 2 above)
Comment (CananOzyol) [reply backend/src/main/resources/application.properties]: thank you I did

File: backend/src/test/java/org/partypets/backend/controller/RandomImageControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        ;
+    }
+
+    @AfterAll
+    static void afterAll() throws IOException {
+        mockWebServer.shutdown();
+    }
+
+
+}
```
[the remark this answers is not in the document]
Comment (CananOzyol) [reply backend/src/test/java/org/partypets/backend/controller/RandomImageControllerTest.java]: thank you I did

**Review by ZeshanShahid79 — APPROVED** [review]
great job!

**Pull request merged**